### PR TITLE
added reversing writeups

### PR DESCRIPTION
**Pull request by simran-kathpalia**

### CODE REVIEW

File: docs/reversing/tamuCTF/intro.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Reversing
```
Comment (sandhrabino) [inline docs/reversing/tamuCTF/intro.md]: No need of intro.md here

Comment (simran-kathpalia) [reply docs/reversing/tamuCTF/intro.md]: okay

**Comment by sandhrabino** [comment]
Remove intro.md in tamuCTF - not needed

File: docs/reversing/tamuCTF/intro.md (hunk 1 above)
Comment (simran-kathpalia) [reply docs/reversing/tamuCTF/intro.md]: i should put it in the outer intro.md?

**Comment by simran-kathpalia** [comment]
Changed now. Check

**Pull request merged**